docs/WHOLE_STUDY_IMPORT.md

```diff
@@ -0,0 +1,204 @@
+## Final output example
+In the end, you'll end up with this example output from `pbta_all` study in `processed` dir:
+```sh
+processed
+└── pbta_all
+    ├── case_lists
+│   ├── cases_3way_complete.txt
+│   ├── cases_RNA_Seq_v2_mRNA.txt
+│   ├── cases_all.txt
+│   ├── cases_cna.txt
+│   ├── cases_cnaseq.txt
+│   ├── cases_sequenced.txt
+│   └── cases_sv.txt
+├── data_CNA.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_cnvs/pbta_all.discrete_cnvs.txt
+├── data_clinical_patient.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_patient.txt
+├── data_clinical_sample.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_sample.txt
+├── data_clinical_timeline_clinical_event.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_timeline_clinical_event.txt
+├── data_clinical_timeline_imaging.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_timeline_imaging.txt
+├── data_clinical_timeline_specimen.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_timeline_specimen.txt
+├── data_clinical_timeline_surgery.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_timeline_surgery.txt
+├── data_clinical_timeline_treatment.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/datasheets/data_clinical_timeline_treatment.txt
+├── data_cna.seg.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_cnvs/pbta_all.merged_seg.txt
+├── data_linear_CNA.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_cnvs/pbta_all.predicted_cnv.txt
+├── data_mutations_extended.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_mafs/pbta_all.maf
+├── data_rna_seq_v2_mrna.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_rsem/pbta_all.rsem_merged.txt
+├── data_rna_seq_v2_mrna_median_Zscores.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_rsem/pbta_all.rsem_merged_zscore.txt
+├── data_sv.txt -> /home/ubuntu/volume/PORTAL_LOADS/pbta_all/merged_fusion/pbta_all.fusions.txt
+├── meta_CNA.txt
+├── meta_clinical_patient.txt
+├── meta_clinical_sample.txt
+├── meta_clinical_timeline_clinical_event.txt
+├── meta_clinical_timeline_imaging.txt
+├── meta_clinical_timeline_specimen.txt
+├── meta_clinical_timeline_surgery.txt
+├── meta_clinical_timeline_treatment.txt
+├── meta_cna.seg.txt
+├── meta_linear_CNA.txt
+├── meta_mutations_extended.txt
+├── meta_rna_seq_v2_mrna.txt
+├── meta_rna_seq_v2_mrna_median_Zscores.txt
+├── meta_study.txt
+└── meta_sv.txt
+```
+Note! Most other studies won't have a timeline set of files.
+## Upload the final packages
+Upload all of the directories named as study short names to `s3://kf-strides-232196027141-cbioportal-studies/studies/`. You may need to set and/or copy your aws saml key before uploading. See "Access to [AWS Infra PedCBioPortal Import](https://github.com/d3b-center/aws-infra-pedcbioportal-import) repo" bullet point in [Software Prerequisites](../README.md#software-prerequisites) to load the study.
```

Comment:
We should probably update this as authentication now done by assuming a role, like this:
```sh
export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s" $(aws sts assume-role --role-arn arn:aws:iam::232196027141:role/bix-dev-data-role --role-session-name MySessionName --query "Credentials.[AccessKeyId,SecretAccessKey,SessionToken]" --output text))
```
I have it set as an alias on my ec2. Then you'd do `aws s3 cp` with no profile required